prowler/providers/kubernetes/kubernetes_provider.py

```diff
@@ -271,7 +270,7 @@ def get_context_user_roles(self):
             logger.critical(
                 f"{error.__class__.__name__}[{error.__traceback__.tb_lineno}]: {error}"
             )
-            sys.exit(1)
+            raise error
```

Comment:
Add a TODO here to replace it with our custom exceptions once they are ready.

Reply:
Done

Reply:
Done